### PR TITLE
[AIRFLOW-5829] Get rid of the checklicence image.

### PR DESCRIPTION
**Pull request by potiuk**
This change is a further step of simplifying the set of scripts
used by CI. The separate checklicence image was implemented as an
optimisation of the licence check time. The image to download was
small and could be downloaded slightly faster in CI. However that
made all the management script more complex and lead to having
separate jobs for check licence and static checks. That lead to
actually longer time of Travis jobs - because new machine had to
be spun-off for checklicence check only.

With this change, the CI image is the only one left and it is slightly
bigger (with RAT tool added) but the same image is used for all the
tests - unit tests, static checks and checklicence checks.

This also makes it easier to manage the images and decreases update
overhead on the developers using Breeze.

Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-5829

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -203,6 +203,20 @@ RUN curl -Lo kind \
    && chmod +x kind \
    && mv kind /usr/local/bin/kind
 
+ARG RAT_VERSION="0.13"
+
+ENV RAT_VERSION="${RAT_VERSION}" \
+    RAT_JAR="/tmp/apache-rat-${RAT_VERSION}.jar" \
```
Comment (mik-laj) [inline Dockerfile]: Can we put it in another directory? For example `/opt/`. The `/tmp/` directory is by definition a ephemeral directory, but we want this file to exist longer.

Comment (potiuk) [reply Dockerfile]: Sure. I will correct it for hive/minicluster and others above.

**Comment by potiuk** [comment]
This one is green. Looking forward to merging this one as well (getting rid of checklicence image).

**Comment by potiuk** [comment]
And we have now one static check job in Travis run with single image - less overhead for starting VMs on Travis. See https://travis-ci.org/apache/airflow/builds/607930228

**Comment by potiuk** [comment]
Note that this build lasts longer but when we merge to master the image will be pushed to Dockerhub (I will speed it up with my build server in GCP) and they will be much faster (10 minutes each).

**Review by mik-laj — APPROVED** [review]

**Pull request merged**